src/renderer/components/settings/Information.tsx

```diff
@@ -47,9 +44,12 @@ export class LanguageSettings extends React.Component<Props, States> {
 
     public async componentDidMount() {
         const { locale } = this.props;
-
+        let filePath: string = path.join((global as any).__dirname, __INFO_MD_RELATIVE_URL__);
```

Comment:
Minor detail: rename `filePath` to `folderPath` (for the avoidance of doubt when reading the source code)